cli/command/load_balancer_command_test.go

```diff
@@ -273,3 +286,86 @@ func TestListLoadBalancers(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestHealthCheck_noUpdateRequired(t *testing.T) {
+	tc, ctrl := newTestCommand(t)
+	defer ctrl.Finish()
+	command := NewLoadBalancerCommand(tc.Command())
+
+	tc.Resolver.EXPECT().
+		Resolve("load_balancer", "env").
+		Return([]string{"id"}, nil)
+
+	tc.Client.EXPECT().
+		GetLoadBalancer("id").
+		Return(&models.LoadBalancer{}, nil)
+
+	c := getCLIContext(t, Args{"env", "name"}, nil)
+	if err := command.HealthCheck(c); err != nil {
+		t.Fatal(err)
+	}
+}
+
+func TestHealthCheck_partialUpdateRequired(t *testing.T) {
+	tc, ctrl := newTestCommand(t)
+	defer ctrl.Finish()
+	command := NewLoadBalancerCommand(tc.Command())
+
+	existingHealthCheck := models.HealthCheck{
+		Target:             "TCP:80",
+		Interval:           30,
+		Timeout:            5,
+		HealthyThreshold:   2,
+		UnhealthyThreshold: 2,
+	}
+
+	tc.Resolver.EXPECT().
+		Resolve("load_balancer", "env").
+		Return([]string{"id"}, nil)
+
+	tc.Client.EXPECT().
+		GetLoadBalancer("id").
+		Return(&models.LoadBalancer{
+			HealthCheck: existingHealthCheck,
+		}, nil)
+
+	expectedHealthCheck := models.HealthCheck{
+		Target:             "TCP:88",
+		Interval:           45,
+		Timeout:            5,
+		HealthyThreshold:   2,
+		UnhealthyThreshold: 2,
+	}
+
+	tc.Client.EXPECT().
+		UpdateLoadBalancerHealthCheck("id", expectedHealthCheck)
+
+	flags := Flags{
+		"set-target":   "TCP:88",
+		"set-interval": 45,
+	}
+
+	c := getCLIContext(t, Args{"env", "name"}, flags)
+	if err := command.HealthCheck(c); err != nil {
+		t.Fatal(err)
+	}
+}
+
+func TestHealthCheck_userInputErrors(t *testing.T) {
+	tc, ctrl := newTestCommand(t)
+	defer ctrl.Finish()
+	command := NewLoadBalancerCommand(tc.Command())
+
+	contexts := map[string]*cli.Context{
+		"Non-int '--set-interval' flag":            getCLIContext(t, Args{"env", "name"}, Flags{"set-interval": "two"}),
```

Comment:
Almost there:
* This command only takes one arg, so these calls should be `getCLIContext(t, Args{"name"}, Flags{...`
* There should be a case for a missing `NAME` arg: `getCLIContext(t, nil, Flags{...`